modular_splurt/code/datums/interactions/lewd_definitions.dm

```diff
@@ -0,0 +1,9 @@
+
+/mob/living/cum(mob/living/partner, target_orifice)
+	. = ..()
+	if(src != partner)
+		if(iswendigo(partner) && partner.pulling == src)
+			var/mob/living/carbon/wendigo/W = partner
+			W.slaves |= src
+			to_chat(src, "<font color='red'> You are now [W]'s slave! Serve your master properly! </font>")
```

Comment:
we already have a modular edit of /mob/living/cum() so it'd be better if you add this to that one. Also, this doesn't take in account climaxing using the normal climax verb, so you'll wanna find a better way to implement this